src/renderers/dom/shared/ReactDOMComponent.js

```diff
@@ -346,7 +345,7 @@ function trapBubbledEventsLocal() {
         if (mediaEvents.hasOwnProperty(event)) {
           inst._wrapperState.listeners.push(
             ReactBrowserEventEmitter.trapBubbledEvent(
-              EventConstants.topLevelTypes[event],
```

Comment:
Since topLevelTypes returned the same value as the key, this was a no-op except for the case where the event is not in the list where it would return undefined.

It turns out that just above we check if the event is in the mediaEvent list ( https://github.com/facebook/react/blob/6a659606415d4853026df6e6d525274fdc7d35ea/src/renderers/dom/shared/ReactDOMComponent.js#L288 ) which is a subset of topLevelTypes, so this should be safe.